Makefile.include

```diff
@@ -327,13 +327,13 @@ APPLICATION := $(strip $(APPLICATION))
 
 ifeq (,$(and $(DOWNLOAD_TO_STDOUT),$(DOWNLOAD_TO_FILE)))
   ifeq (,$(WGET))
-    ifeq (,$(shell command -v wget))
```

Comment:
this should have been a neq

Reply:
This is a revert PR. Not touching anything here.

Reply:
Good catch, will process in the followup second attempt.